bot.js

```diff
@@ -17,7 +17,7 @@ const chance = new Chance();
 var Botkit = require('botkit/lib/Botkit.js');
 
 var controller = Botkit.slackbot({
-  debug : process.env.SLACKBOT_DEBUG
+  debug : process.env.SLACKBOT_DEBUG == 'TRUE'
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/ondra/obed-o-bot/file/2229873906/issues/source?bid=3165659&fileBranchId=3175486#l20)